source/use-cases/sensitive-data-encryption.txt

```diff
@@ -300,6 +289,17 @@ information:
 - The BSON Type of each field (only required for deterministically
   encrypted fields)
 
+.. admonition::JSON Schema Validation
+   :important:
+   
+   MongoDB drivers only use JSON Schema to configure automatic client-side
```

Comment:
The drivers use the JSON Schema *syntax* to configure automatic client-side field level encryption and support *only* field level encryption-specific keywords.

Reply:
Thanks, this detail really helps clarify the point. Will update.